text/trace/0235-sampling-threshold-in-trace-state.md

```diff
@@ -80,23 +102,54 @@ The `T` value MUST be derived as follows:
 
 Sampling Decisions MUST be propagated by setting the value of the `th` key in the Tracestate header according to the above.
 
-## Initializing and updating T and R values
+This design requires that as a given span progresses along its collection path, `th` is non decreasing (and, in particular, must be increased at stages that apply lower sampling probabilities).
```

Comment:
This paragraph and the next one seem to imply that increasing T/th is fine, but decreasing it is not, whereas the Head Samplers sections puts this as an actual example.

Reply:
Ping @kalyanaj